### DISCUSSION TITLE
Dynamic Gets

### DISCUSSION THREAD

**Discussion by Safweb** [joomengine/Joomla-Component-Builder#941] [Q&A]
I have made a custom Dynamic Gets and added it to a front page display, but it comes with the result of the query at the top of the page and I can't get it to appear in the table, I think I've tried everything. Is there someone who can help me?

![fejl_jcb](https://user-images.githubusercontent.com/33495847/204544498-3b9762a3-8a92-4b4b-a962-8cdefd5415c3.png)

**Comment by Llewellynvdm** [comment]
Okay this question is way to broad, you will have to explain more of what your doing, and what your trying to achieve.
